fuzz/client.c

```diff
@@ -25,6 +26,23 @@ extern int rand_predictable;
 /* unused, to avoid warning. */
 static int idx;
 
+#define FUZZTIME 1485898104
+
+#ifdef _WIN32
+# ifdef _TIME64_T_DEFINED
+time64_t _time64(time64_t *t)
+# else
+time_t _time(time_t *t)
```

Comment:
There is no `_time` symbol in msvcrt.dll. There normally is `_time64`, `_time32` and `time`. One probably should keep in mind that 32-bit symbols are decorated, i.e. if you enumerate symbols in 32-bit dll, you will note `_time`, *but* from C it's referred to as `time`, not `_time`. I also want to remind that suggested snippet took "*opportunistic*" approach, it's not very accurate. Idea is balance #ifdef "spaghetti" vs. coverage of most common cases. Or in other words something that would provide higher chance of performing the trick would have more #if conditions. I'd say that if one wants to take if to next level it would rather be
```
#if defined(_WIN32) && defined(_TIME64_T_DEFINED)
time64_t _time64(time64_t *t) { implementation }
#endif
#if !defined(_WIN32) || !defined(_MSC_VER)
time_t time(time_t *t) { implementation }
#endif
```
I.e sometimes *two* implementations with different names. Formally speaking `#if !defined(_WIN32) || !defined(_MSC_VER)` can be reduced to `#if !defined(_MSC_VER)`, but former is probably more informative.